public/index.html

```diff
@@ -248,7 +248,7 @@ <h2>Routes</h2>
 
       <p>
         <strong>Note</strong>: you can view detailed examples
-        <a href="guide.html">here</a>.
+        <a href="/guide.html">here</a>.
```

Comment:
Good catch, thanks. I think you meant

```suggestion
        <a href="/guide/">here</a>.
```